.tekton/visual-qontract-master-pull-request.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: tekton.dev/v1
```

Comment:
Lets rename the file too -> `main-pull` instead of `master-pull`

Reply:
done